tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_custom_StyledText.java

```diff
@@ -5540,13 +5541,19 @@ protected void rtfCopy() {
 	text.addLineStyleListener(listener);
 	linesCalled[0] = 0;
 	text.copy();
-	assertEquals("not all lines tested for RTF copy", text.getLineCount(), linesCalled[0]);
+
+	// The listener is invoked twice for each line, once for RTF and once for HTML.
+	assertEquals("not all lines tested for RTF & HTML copy", 2 * text.getLineCount(), linesCalled[0]);
```

Comment:
Can't you just copy this test for HTML as well? I believe trying to test RTF and HTML at the same time is not likely to make things easier.

Reply:
Ah ok, got it, both transfers are computed on cut/copy; we cannot separate the computation.

Reply:
ACK